test.js

```diff
@@ -1,9 +1,21 @@
 import test from 'ava';
 import alfyTest from 'alfy-test';
+import pidFromPort from 'pid-from-port';
 
-test(async t => {
+test('search by name', async t => {
 	const alfy = alfyTest();
 	const result = await alfy('bash');
 
 	t.true(result.length > 0);
 });
+
+test('search by port', async t => {
+	const pids = await pidFromPort.list();
+	const ports = [...pids.keys()];
```

Comment:
If you only want the first one you could do `const [port] = [...pids.keys()];`

Reply:
Good point :)! It doesn't seem to work on Travis though so might have to remove it. Or disable it on CI.

Reply:
Maybe create a fixture with a port and try searching for that? Could be that `ports` here is empty on Travis.

Reply:
Oh yes! Great idea!